src/components/UI/PasswordInput.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="wrapper">
+    <label for="password" class="password-label">Password</label>
+    <input
+      class="password-input"
+      required
+      :type="inputType"
+      placeholder="Enter your password"
+      :value="value"
+      name="password"
+      @input="$emit('input', $event.target.value)"
+      @blur="$emit('blurTouched')"
+    />
+    <button class="password-eye" @click.prevent="toggleViewPassword">
+      <img src="../../assets/img/eye.png" alt="eye" />
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "PasswordInput",
+  props: {
+    value: {
+      type: String,
+      default: "",
+    },
+  },
+  data() {
+    return {
+      showPassword: false,
+    };
+  },
+  methods: {
+    toggleViewPassword: function () {
```

Comment:
```suggestion
    toggleViewPassword() {
```

it is exactly the same, but shorter and a bit more consistent with other places 😉